sdks/java/core/src/main/java/org/apache/beam/sdk/io/WriteFiles.java

```diff
@@ -133,37 +133,50 @@
   private final ValueProvider<Integer> numShardsProvider;
   private final boolean windowedWrites;
   private int maxNumWritersPerBundle;
+  // If set,this is used instead of DynamicDestinations.getSideInputs. This is used by the Beam
+  // translation code.
+  private final List<PCollectionView<?>> overriddenSideInputs;
+
+  private List<PCollectionView<?>> getSideInputs() {
+    return (overriddenSideInputs != null)
+        ? overriddenSideInputs
+        : getSink().getDynamicDestinations().getSideInputs();
```

Comment:
I'm confused about what you're asking - are simply suggesting that I change the comment, or do something else?